src/group_theory/subgroup.lean

```diff
@@ -715,6 +715,18 @@ lemma comap_infi {ι : Sort*} (f : G →* N) (s : ι → subgroup N) :
 @[simp, to_additive] lemma comap_top (f : G →* N) : (⊤ : subgroup N).comap f = ⊤ :=
 (gc_map_comap f).u_top
 
+@[to_additive]
+lemma eq_bot_iff_map_eq_bot {G' : Type*} [group G'] {f : G →* G'} (hf : function.injective f)
+  (H : subgroup G) : H = ⊥ ↔ H.map f = ⊥ :=
```

Comment:
I would be inclined to swap the two sides. In general, it's best to put the simplest expression on the RHS.

Reply:
Done.